scripts/sonoran_desert.json

```diff
@@ -0,0 +1,213 @@
+{
+    "citation": "",
+    "description": "Long-term population dynamics of individually mapped Sonoran Desert winter annuals from the Desert Laboratory, Tucson AZ",
+    "encoding": "ISO-8859-1",
+    "homepage": "http://www.eebweb.arizona.edu/faculty/venable/LTREB/LTREB%20data.htm",
+    "keywords": [
+        "long-term population dynamics",
+        "demography",
+        "Sonoran Desert",
+        "annual plants",
+        "seed bank"
+    ],
+    "name": "sonoran-desert",
+    "resources": [
+        {
+            "dialect": {
+                "contains_pk": "False",
+                "delimiter": ","
+            },
+            "name": "species_per_year",
```

Comment:
Since the species_per_year table has  dots (**`.`**) as empyt/None/Null values, let us tell the retriever to give an appropriate value for those dots. Add `"missingValues": ["."]` in the dialect
```j
"dialect": {
    "contains_pk": "False",
     "missingValues": ["."]
```

After, try to install both old and new version and check the difference.
You can add more values, if they are available, to the list `missingValues": [".", "", "N?A" ]`